core/src/main/java/org/infinispan/topology/ClusterCacheStatus.java

```diff
@@ -172,6 +184,21 @@ private boolean addMember(Address joiner, CacheJoinInfo joinInfo) {
             return false;
          }
 
+         // Validate if the member is allowed to join
+         if (persistentState.isPresent()) {
```

Comment:
So the cache's persistent state must already exist on the coordinator? What if one of the nodes' disk is lost, and that happens to be the first node to start?